sudoku-logic/src/test/java/am/iunetworks/training/sudoku/logic/GameManipulationsHelperTest.java

```diff
@@ -55,21 +55,49 @@ public void testBoardRotate() {
 	}
 
 	@Test
-	public void testSwitchRowsAndColumns() {
-		//board
+	public void testSwitchRows() {
+		// board
 		String testBoard = "375964182964182375182375964759641823641823759823759641596418237418237596237596418";
+		try {
+			new GameBoardManipulationsHelper(testBoard).switchRows(10, 10);
+			fail("Helper should throw NPE if initialized with null");

```

Comment:
copy-paste-ը լավ բանա, բայց պետքա ուշադիր լինել => "Helper should throw **NPE if initialized with null**"